File: src/03/bubblesort/bubblesort.js
function bubbleSort(array) {
    const len = array.length;

    for (let i = 0; i < len - 1; ++i) {
        for (let j = 0; j < len - i -1; j++) {
            if (array[j] > array[j+1]) {
                [array[j+1], array[j]] = [array[j], array[j+1]];
            }
        }
    }

    return array;
}

const array = [1, 3, 2, 6, 5, 7, 8, 9, 10, 0]
console.log(bubbleSort(array));

File: src/03/bubblesort/bubblesort.py
def bubbleSort(arr):
    for i in range(0, len(arr)):
        for j in range(0, len(arr) - i - 1):
            if arr[j] > arr[j+1]:
                arr[j], arr[j + 1] = arr[j + 1], arr[j]
    return arr

arr = [1,3,44,6,8,10,9]
print(bubbleSort(arr))
